src/com/lightdev/app/shtm/SplitPanel.java

```diff
@@ -123,23 +112,18 @@ private void buildLayout() {
     public void removeAllOuterPanels() {
         // Warning: it does not really remove the outer panels per se.
         JComponent p;
-        if (majorAxis == MAJOR_AXIS_VERTICAL) {
```

Comment:
Why have you removed this if statement?

Reply:
at initialization
```
public static final int MAJOR_AXIS_VERTICAL = 2;
...
private final int majorAxis = SplitPanel.MAJOR_AXIS_VERTICAL;
```
both variables will always have the same value, only the if block will be executed always

Reply:
Do you think we should also remove MAJOR_AXIS_HORIZONTAL "NOT SUPPORTED" ?

Reply:
yeah, I think we can remove
MAJOR_AXIS_HORIZONTAL
MAJOR_AXIS_VERTICAL
majorAxis
none of them are being used. I can do it later today